src/status_im/ui/screens/browser/styles.cljs

```diff
@@ -62,29 +62,38 @@
 (def web-view-error-text
   {:color colors/gray})
 
-(defnstyle toolbar-content [show-actions]
+(def toolbar-content
   {:flex-direction     :row
    :flex               1
-   :border-radius      4
-   :height             36
+   :border-radius      8
+   :max-height         36
    :background-color   colors/gray-lighter
    :padding-horizontal 12
-   :margin-right       5
+   :margin-right       8
    :align-items        :center
-   :android            {:margin-left (if show-actions 66 20)}
-   :ios                {:margin-left 20}})
+   :align-self         :center
+   :margin-top         8
+   :margin-left        56})
 
 (defstyle url-input
-  {:flex              1
-   :font-size         14
-   :letter-spacing    -0.2
-   :margin-horizontal 5
-   :android           {:padding 0}})
+  {:flex                1
+   :font-size           15
+   :line-height         22
+   :letter-spacing      -0.2
```

Comment:
guys please, don't use letter-spacing property on text, thank you 🙏

Reply:
never? there is 99 use of letter-spacing in the code base should I remove them all?

Reply:
that would make my day

Reply:
letter spacing was, kinda, required when using the system font on iOS to make it look native, iOS uses letter spacing in native apps. But we don't use system fonts anymore and we're not trying to imitate the exact native 'look' :)

Reply:
or you can leave it for now, and return to it once we introduce text styles as variables